### DISCUSSION TITLE
Team meeting planning discussion for 2022-03-02

### DISCUSSION THREAD

**Discussion by nalininsbs** [wpmarketingteam/Marcomms-Internal-Comms#19] [General]
Please add any suggestions for the agenda to this card. Thank you. We are testing the access to the discussion area in the new central marketing team repository. Once this is confirmed as working, we will move this weekly meeting discussion there. Thank you

**Comment by nalininsbs** [comment]
_Starting a draft._ I have made slight adjustments to some of the headings to better incorporate asynchronous contribution and to make it easier for the notetakers in a template. Some areas of the notes which have set text can use blocks going forward. This will also save time for notetakers and reviewers.

Items in double brackets below are to incorporate into the script.

DRAFT AGENDA

1. Welcome, Introduction
((- include link to agenda))
((- encourage contribution))

2. Getting to know each other  ((Update sub head in the script))
2a) Meeting attendees and async contributors
2b) Welcoming new arrivals and returners to the channel
2c) Social question of the week: “What is your favorite hobby or pastime? Please share why you would encourage someone else to try it too. 

3) Notes from the last meeting

4) Projects and task updates
a) Newsletters (what will we have coming up this week and next week)

b) GitHub link and updates
<a href="https://github.com/orgs/WordPress/projects/21">[main marketing board](https://github.com/orgs/WordPress/projects/21)

c) WordCamp Europe Contributor Day

d) Learn WordPress
((Key links for this week))

e) WP Diversity 

f) Contributions to next week's marketing planning discussion ((link at the end again too))
[Next week's Marketing Team meeting planning discussion](https://github.com/WordPress/Marcomms-Internal-Comms/discussions) ((CHECKING LINK GOES TO NEXT WEEK'S))
((Depending on the access test, this may still be on the internal comms repo this week with an invitation to contributors to help test it in is new location))

5) Important dates
- next coffee break date
- check out the team calendar with dates for meetings, collaborations and social activities [Google calendar] (https://calendar.google.com/calendar/b/2?cid=NDQyN2g1cnZrN2EwYXA5Y3VpNDI5OGYzY3NAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
- next meeting: Wednesday 2 March 2022 at 15:00 UTC. You are very welcome to contribute to it asynchronously.   

6) Open floor

**Comment by nalininsbs** [comment]
**Links for awareness:**
Proposal for a news blog on developer.wordpress.org from @bph 
https://make.wordpress.org/core/2022/02/25/proposal-to-start-a-news-blog-on-developer-wordpress-org/

Project roadmap
https://wordpress.org/about/roadmap/

**Comment by meherbala** [comment]
**Links for awareness:**

People of WordPress: Tonya Mork:   https://wordpress.org/news/2022/02/people-of-wordpress-tonya-mork/

**Reply by murillol** [reply]
Thanks Meher!

**Comment by murillol** [comment]
Hi! I'm confused, are we using this space to plan, or this one: https://github.com/WordPress/Marketing-Team/discussions

I didn't see this entire thread until today because I thought we were using the other one?!?

**Reply by nalininsbs** [reply]
Hello. From the 9 March 2022, we are testing the area in the new central repository to check whether contributors can access it. If not, we will need to continue to use the one in the internal communications repository, and keep that open.. Thank you

**Reply by murillol** [reply]
ooh ok! 
Also, I can't close any issues anywhere in this repo (you asked me to close one, I think the WPDiversity issue, but the option just doesn't appear for me)
